tpp-app/src/services/CalendarService.js

```diff
@@ -0,0 +1,120 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { KEYS, TRACK_SYMPTOMS } from './utils/constants.js'
+import { Symptoms } from './utils/models.js';
+
+/**
+ * Makes API request to AsyncStorage to access symptoms for the entire year
+ * @param year: number
+ * @returns: object
+ */
+export const GETYearData = async (year) => {
```

Comment:
Think there is some overlap with the function `getCalendarByYear` I wrote in my PR, but I think it's fine. There's a difference, namely that the method I wrote gets the current calendar, & the prev & next year's calendar if it exists. It's currently not converting all the items to symptoms, but I can swap it so it calls this function to make functionality consistent.

tldr: I have a similar function, but it gets multiple years. I can modify it to use this func and we will be chilling imo